pubspec.lock

```diff
@@ -934,26 +944,28 @@ packages:
   maplibre_gl:
     dependency: "direct main"
     description:
-      path: "../flutter-maplibre-gl"
-      relative: true
-    source: path
+      path: "."
+      ref: "0bf326044759dfc2f590a1c219d2f1eb86e7b8ad"
+      resolved-ref: "0bf326044759dfc2f590a1c219d2f1eb86e7b8ad"
+      url: "https://github.com/YumNumm/flutter-maplibre-gl.git"
+    source: git
```

Comment:
Seems pubspec.lock was outdated (overriden somewhere?)

Reply:
I forgot to add `pubspec_overrides.yaml`. 

* added `pubspec_overrides.yaml` at 9a03428
* executed `flutter pub get` at ea5ff41